lib/cfa/augeas_parser.rb

```diff
@@ -119,12 +155,20 @@ def []=(key, value)
       end
     end
 
+    # @param matcher [Matcher]
+    # @return [Array<AugeasElement>] matching elements
     def select(matcher)
       @data.select(&matcher)
     end
 
     # @note for internal usage only
     # @private
+    #
```

Comment:
well, it is intention. `--private` is for private methods. but `@private` is marking method as really internal that should be excluded from documentation.